Packs/CortexXDR/TestPlaybooks/Test_XDR_Playbook.yml

```diff
@@ -1,24 +1,22 @@
 id: Test XDR Playbook
-version: -1
+version: 2
 name: Test XDR Playbook
-fromversion: 4.1.0
```

Comment:
this playbook is using the new playbook with is from version 5.5 and you removed this line.
currently, since the is no `fromversion` property in the test, it is loaded (unsuccessfully to all demisto versions).
add `fromversion` in this file.
any reason the new playbook is only from version 5.5 and not earlier?

Reply:
also, add `fromversion` in the `conf.json` file as the test should not run on all versions.

Reply:
it was for testing reasons. I've converted it back. thanks.